tests/testthat/_snaps/num.md

```diff
@@ -48,50 +48,111 @@
       notation = "si"), )
     Output
       # A tibble: 20 x 4
-              sci        eng                   dec         si
-            <sci>      <eng>                 <dec>       <si>
-       1 1.00e-13 100.00e-15       0.0000000000001    100.00f
-       2 1.00e-12   1.00e-12       0.000000000001       1.00p
-       3 1.00e-11  10.00e-12       0.000000000010      10.00p
-       4 1.00e-10 100.00e-12       0.0000000001       100.00p
-       5 1.00e- 9   1.00e- 9       0.000000001          1.00n
-       6 1.00e- 8  10.00e- 9       0.00000001          10.00n
-       7 1.00e- 7 100.00e- 9       0.0000001          100.00n
-       8 1.00e- 6   1.00e- 6       0.000001             1.00µ
-       9 1.00e- 5  10.00e- 6       0.00001             10.00µ
-      10 1.00e- 4 100.00e- 6       0.0001             100.00µ
-      11 1.00e- 3   1.00e- 3       0.001                1.00m
-      12 1.00e- 2  10.00e- 3       0.01                10.00m
-      13 1.00e- 1 100.00e- 3       0.1                100.00m
-      14 1.00e+ 0   1.00e+ 0       1                    1.00 
-      15 1.00e+ 1  10.00e+ 0      10                   10.00 
-      16 1.00e+ 2 100.00e+ 0     100                  100.00 
-      17 1.00e+ 3   1.00e+ 3    1000                    1.00k
-      18 1.00e+ 4  10.00e+ 3   10000                   10.00k
-      19 1.00e+ 5 100.00e+ 3  100000                  100.00k
-      20 1.00e+ 6   1.00e+ 6 1000000                    1.00M
+            sci      eng                   dec       si
+          <sci>    <eng>                 <dec>     <si>
+       1 1.e-13 100.e-15       0.0000000000001    100.f
```

Comment:
`1.e` looks odd to my eye, and `1 e` is just as weird.

Reply:
Fixing in a forthcoming PR.